include/perf_watcher.h

```diff
@@ -9,8 +9,14 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+typedef enum {
+  PWRequestedFalse = 0, // always off
```

Comment:
Wow, we have 3 different conventions on enums, and I managed to be inconsistent with all of them.